### PR TITLE
Build Debian based Prince 13.1 images

### PR DESCRIPTION
**Pull request by wezm**
Dockerfiles and Makefile for building Prince Docker images based on `debian-{8,9,10}` and `debian-{8,9,10}-slim` images. Other images can be added as needed.

The Makefile is a bit repetitious. I'm open to improvements/recommendations on how to make it nicer.

### CODE REVIEW

File: Dockerfile.debian [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      fonts-lohit-gujr \
+      fonts-lohit-guru \
+      fonts-lohit-beng-bengali \
+      fonts-lohit-deva \
+      fonts-baekmuk \
+      fonts-ipafont-mincho \
+      fonts-arphic-uming \
+      fonts-opensymbol \
+      fonts-liberation2 && \
+    curl --proto '=https' --tlsv1.2 -O https://www.princexml.com/download/${DEB_FILE} && \
```
Comment (mikeday) [inline Dockerfile.debian:27]: Are the proto and tlsv1.2 arguments necessary for security?

**Comment by wangp** [comment]
Maybe keep ca-certificates and replace curl-ca-bundle.crt with a symlink to /etc/ssl/certs/ca-certificates.crt? It's arguable that the .deb should do that itself.

**Review by mikeday — APPROVED** [review]
Looks okay to me! @wangp might want to take a quick look at the dependencies.

File: Dockerfile.debian (hunk 1 above)
Comment (wezm) [reply Dockerfile.debian:27]: It makes `curl` use https and and TLS1.2 or newer. This helps ensure that it actually uses https and doesn't fall back to an insecure TLS version — failing to do either of these things would suggest something is a bit fishy and warrants further investigation.

**Pull request merged**